static/src/javascripts-legacy/projects/common/modules/check-dispatcher.js

```diff
@@ -43,34 +43,30 @@ define([
             return config.switches.emailInArticleOutbrain;
         },
         hasHighPriorityAdLoaded: function() {
-            return checkMediator.waitForCheck('thirdPartyTagsDisabled').then(function (thirdPartyTagsDisabled) {
-                // if thirdPartyTagsDisabled no external ads are loaded
-                if (thirdPartyTagsDisabled) {
-                    return false;
-                } else {
-                    return trackAdRender('dfp-ad--merchandising-high').then(function(highPriorityAdLoaded) {
-                        return highPriorityAdLoaded;
-                    });
-                }
-            });
+            // if thirdPartyTags false no external ads are loaded
+            if (commercialFeatures.thirdPartyTags) {
+                return trackAdRender('dfp-ad--merchandising-high').then(function(highPriorityAdLoaded) {
+                    return highPriorityAdLoaded;
```

Comment:
Yep, same for trackAdRender('dfp-ad--merchandising') further down in hasLowPriorityAdLoaded, I'll update both.